packages/backend/src/accounting/service.ts

```diff
@@ -139,6 +163,9 @@ export async function createAccountToAccountTransfer(
         return error
       }
 
+      await onDebit(sourceAccount)
```

Comment:
do these need to happen sequentially? If not, maybe awaiting a Promise.all would be an improvement.